### PR TITLE
Bump to kube 1.16

### PR DESCRIPTION
**Pull request by ecordell**
**Description of the change:**

This bumps the kube libraries to the versions used in kube 1.16.

It also adds one supporting method to the bundle `csv` abstraction so that OLM can get the version data without fully parsing the csv from the bundle.

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: pkg/registry/csv.go [hunk 1]
```diff
@@ -66,6 +69,28 @@ func (csv *ClusterServiceVersion) GetReplaces() (string, error) {
 	return replaces, nil
 }
 
+// GetVersion returns the version of the CSV
+//
+// If not defined, the function returns an empty string.
+func (csv *ClusterServiceVersion) GetVersion() (string, error) {
```
Comment (ecordell) [inline pkg/registry/csv.go]: This is the only non-dependency change

**Comment by jpeeler** [comment]
I'm curious how to properly solve this:

vendor/github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient/serviceaccount.go:38:2: not enough arguments to return
vendor/github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient/serviceaccount.go:38:10: c.Core undefined (type *Client has no field or method Core)

The first OLM release to not use c.Core is 4.2. Perhaps the rule should be to vendor the latest OLM that contains the most recent rebase?

**Comment by dinhxuanvu** [comment]
@ecordell I assume you remove the schema because we have a validation library now and will vendor that library at a later time?

**Comment by ecordell** [comment]
@dinhxuanvu Yep! I mentioned that in the commit message as well.

**Comment by ecordell** [comment]
/hold 

Just noticed the version of olm being vendored is wrong.

**Comment by ecordell** [comment]
/hold cancel

operator-registry no longer depends on OLM at all 🙃

**Pull request merged**